specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3339,12 +3342,29 @@
           "type": "integer",
           "default": 60,
           "description": "Monitoring interval in seconds."
+        },
+        "endpoints": {
+          "type": "array",
+          "description": "List of connection monitor endpoints.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpoint"
+          }
+        },
+        "testGroups": {
+          "type": "array",
+          "description": "List of connection monitor test groups.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorTestGroup"
+          }
+        },
+        "testConfigurations": {
+          "type": "array",
+          "description": "List of connection monitor test configurations.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorTestConfiguration"
+          }
```

Comment:
what is the behavior if the client uses old api version  (that doesn't have these ) ?
It is safe to introduce a new api version.